src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's contacts in the address book.
```

Comment:
this is good, but try to avoid specifics in the header comment. make it as generic as possible.